philadelphia/src/main/java/com/paritytrading/philadelphia/FIXSession.java

```diff
@@ -583,6 +573,15 @@ public void message(FIXMessage message) throws IOException {
             }
         }
 
+        private void handleTooLowMsgSeqNum(FIXMessage message, long msgSeqNum, FIXValue msgType) throws IOException {
```

Comment:
Another small nit: Here I would prefer `msgType` before `msgSeqNum` in the argument list as that is their order in the FIX message header.